python-sdk/langwatch/src/langwatch/utils/exceptions.py

```
@@ -0,0 +1,13 @@
+import traceback
+
+from langwatch.domain import ErrorCapture
+
+
+def capture_exception(err: BaseException):
+    try:  # python < 3.10
+        string_stacktrace = traceback.format_exception(
+            etype=type(err), value=err, tb=err.__traceback__
+        )  # type: ignore
+    except:  # python 3.10+
```

Ruff (0.8.2)

Do not use bare `except`

(E722)